README.md

```diff
@@ -192,74 +192,87 @@ This request performs decoding on an encoded document.
 
-#### encryptString
+### encryptString
 
 This request encrypts a given string with symmetric AES.
 
-##### Parameters
+#### Parameters
 
 This request supports the parameters below:
-<!--Flag: pls check the necessity column, if it is optional or required.-->
 
 | Parameter Name  | Data Type  | Necessity | Description |
 |-----------------|------------|-----------|------------|
-|`document`|**[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)**| Optional | Input a string to encrypt it |
+|`document`|**[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)**| Required | Input a string to encrypt it |
 | `key`  | **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** | Optional | Input an encryption key |
 
-##### Results
+#### Results
+
+`encryptString` returns the `IEncryptionResults`, which contains:
 
-`IEncryptionResults` returns cipherText cipher text in base64:
+- **any** `cipherText` in base64
 
-- Returns **any** `iv` in base64
+- **any** `iv` in base64
 
-- Returns **any** `tag`, which is the authenticated encryption tag in base64
+- **any** `tag`, which is the authenticated encryption tag in base64
 
-- Returns **any** `key`, which is the encryption key in hexadecimal
+- **any** `key`, which is the encryption key in hexadecimal
 
-- Returns **any** `type`, which is the encryption algorithm identifier
+- **any** `type`, which is the encryption algorithm identifier
 
-#### decryptString
+### decryptString
 
 This request decrypts a given ciphertext and its associated variables.
 
-##### Parameters
+#### Parameters
 
 The list below shows the relationship between parameters:
 
 *   `$0` **[IEncryptionResults](#iencryptionresults)**
 
     *   `$0.cipherText` &#x20;
-    *   `$0.tag` &#x20;
     *   `$0.iv` &#x20;
+    *   `$0.tag` &#x20;  
     *   `$0.key` &#x20;
     *   `$0.type` &#x20;
 
 This request supports the parameters below:
-<!--Flag: pls check the necessity column, if it is optional or required.-->
 
 | Parameter Name  | Data Type  | Necessity | Description |
 |-----------------|------------|-----------|------------|
-|`cipherText`|**[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)**| Optional | Cipher text base64 encoded |
-| `tag`  | **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** | Optional | AES authentication tag base64 encoded |
-| `iv` |**[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)**| Optional | IV base64 encoded |
-| `key`  | **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** | Optional | Decryption key hexademical encoded |
+|`cipherText`|**[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)**| Required | Cipher text base64 encoded |
```

Comment:
```suggestion
|`object`|**[IEncryptionResults](#iencryptionresults)**| Required | IEncryptionResults object with the corresponding values |
```

Reply:
updated

Reply:
this is to replace all the existing parameters not just this one, the method itself accepts an object that includes the cipherText, iv, etc.

Reply:
Ok updated